### PR TITLE
[#176089221] Add channel and channel_descr attributes to GetBPDPaymentInstrument

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

File: openapi/index.yaml [hunk 1]
```diff
@@ -165,6 +184,8 @@ definitions:
       - hpan
       - activation_periods
       - enabled
+      - channel
+      - channel_descr
```
Comment (BurnedMarshal) [inline openapi/index.yaml:188]: Are these fields always present? We can put it optional to avoid decoding errors

**Review by BurnedMarshal — COMMENTED** [review]
lgtm apart the comment before

File: openapi/index.yaml (hunk 1 above)
Comment (pasqualedevita) [reply openapi/index.yaml]: ok done, maybe

File: GetBPDPaymentInstrument/handler.ts [hunk 2]
```diff
@@ -83,7 +86,9 @@ function toApiPaymentInstrumentDetail(
           {
             ...pi,
             cancellation: pi.cancellation?.toISOString(),
-            channel_descr: getChannel(pi.channel),
+            channel_descr: fromNullable(pi.channel)
```
Comment (BurnedMarshal) [inline GetBPDPaymentInstrument/handler.ts:89]: This must be removed, we are here inside `activation_periods`

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: GetBPDPaymentInstrument/handler.ts (hunk 2 above)
Comment (pasqualedevita) [reply GetBPDPaymentInstrument/handler.ts]: done :)

**Review by BurnedMarshal — APPROVED** [review]

**Pull request merged**